alfresco-transform-pdf-renderer/alfresco-transform-pdf-renderer/LICENSES.md

```diff
@@ -0,0 +1,4 @@
+### Licenses
```

Comment:
File rename/move ? Same as engine_config.json earlier ... ?

Reply:
Yes, as before. No changes have been made to this file.

Reply:
I assume line-endings only ?